### PR TITLE
Split processing into 2 scripts

### PR DESCRIPTION
**Pull request by sandrinebedard**
## Description
The purpose of this PR is to separtate processing from current  `process_data.sh` into two scripts:
* `preprocess_data.sh`:
    * Manages resampling, reorientation to RPI and gradient distorsion correction
    * The ouput of preprocessing will be the input data for processing (and the new dataset)
* `process_data.sh` (a revised version):
    * Proceeds to segmentation, vertebral labeling, computes cord CSA

Documentation was modified according to the new preprocessing step.
** Both scripts were tested on 2 and 30 subjects, no problem was noticed!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,11 @@ To convert the DICOM dataset in a BIDS structure for this project, run the follo
...
 curate_project.py -path-in <path_DICOM_dataset> -path-output <path_BIDS_dataset>
 ~~~
-The BIDS datset is (for now) in `duke: temp/uk_biobank_BIDS`
+The BIDS datset is (for now) in **|TODO : update with raw and processed repo**
 
-Here is an example of the BIDS data structure of UK biobank data:
+Here is an example of the BIDS data structure of UK biobank dataset:
 ~~~
-uk_biobank_BIDS
+uk-biobank
```
Comment (jcohenadad) [inline README.md:27]: shouldn't it be called `uk-biobank-processed`? (ie: the repos that contains the derivatives)

File: README.md [hunk 2]
```diff
@@ -20,11 +20,11 @@ To convert the DICOM dataset in a BIDS structure for this project, run the follo
 ~~~
 curate_project.py -path-in <path_DICOM_dataset> -path-output <path_BIDS_dataset>
 ~~~
-The BIDS datset is (for now) in `duke: temp/uk_biobank_BIDS`
+The BIDS datset is (for now) in **|TODO : update with raw and processed repo**
```
Comment (jcohenadad) [inline README.md:23]: ah! maybe that TODO addresses my comment below https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/pull/30/files#r565672217

**Review by jcohenadad — APPROVED** [review]
nice 👌

File: README.md (hunk 2 above)
Comment (sandrinebedard) [reply README.md:23]: haha yes! I am waiting for when we officially have  both datasets to update the info about them!

**Pull request merged**